src/components/user/user.service.ts

```diff
@@ -126,19 +126,17 @@ export class UserService {
     await this.userRepo.updateProperties(user, simpleChanges);
 
     // Update email
-    if (email !== undefined) {
-      try {
-        await this.userRepo.updateEmail(user, email);
-      } catch (e) {
-        if (e instanceof UniquenessError && e.label === 'EmailAddress') {
-          throw new DuplicateException(
-            'person.email',
-            'Email address is already in use',
-            e,
-          );
-        }
-        throw new ServerException('Failed to create user', e);
+    try {
```

Comment:
@CarsonF - I've talked about this with Andre and if you look at Task 0736 in Monday you'll see that the issue is dealing with being able to clear out an email once it exists.  I do think that @atGit2021's solution will work, however we wanted to make sure this is an okay thing to do here.  If we make this change, there's a potential that a user (or someone with access) could remove an email and then be locked out (or lock someone else) out of the system if they tried to login.  The fix could be an easy help desk ticket and an admin could add it back in, but it could spiral.  The user could try to register a new account with the old email address, which would work, but the new account obviously wouldn't be tied to the old one and then activity could be split on them...etc.

Any thoughts on this?

Reply:
@CarsonF @bryanjnelson, another issue that would need to be discussed;  this will also impact the reset password functionality.